index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1>Hello Microverse!</h1>
```

Comment:
Kindly add a `class=header` to apply the style in `style.css` because it's included in the requirements:

![class](https://user-images.githubusercontent.com/95412943/212023755-a608d574-6664-474d-83df-a058a837a8d2.PNG)